optimum/habana/diffusers/pipelines/controlnet/pipeline_stable_video_diffusion_controlnet.py

```diff
@@ -0,0 +1,730 @@
+# Copyright 2024 The HuggingFace Team. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import time
+from math import ceil
+from typing import Callable, Dict, List, Optional, Union
+
+import PIL.Image
+import torch
+from diffusers.image_processor import VaeImageProcessor
+from diffusers.models import AutoencoderKLTemporalDecoder
+from diffusers.pipelines.stable_video_diffusion import StableVideoDiffusionPipeline
+from diffusers.pipelines.stable_video_diffusion.pipeline_stable_video_diffusion import (
+    _append_dims,
+    _resize_with_antialiasing,
+)
+from diffusers.schedulers import EulerDiscreteScheduler
+from diffusers.utils import logging
+from diffusers.utils.torch_utils import randn_tensor
+from transformers import CLIPImageProcessor, CLIPVisionModelWithProjection
+
+from ....transformers.gaudi_configuration import GaudiConfig
+from ....utils import speed_metrics
+from ...models import ControlNetSDVModel, UNetSpatioTemporalConditionControlNetModel
+from ..pipeline_utils import GaudiDiffusionPipeline
+from ..stable_video_diffusion.pipeline_stable_video_diffusion import (
+    GaudiStableVideoDiffusionPipeline,
+    GaudiStableVideoDiffusionPipelineOutput,
+)
+
+
+logger = logging.get_logger(__name__)  # pylint: disable=invalid-name
+
+
+class GaudiStableVideoDiffusionPipelineControlNet(GaudiStableVideoDiffusionPipeline):
```

Comment:
```suggestion
class GaudiStableVideoDiffusionControlNetPipeline(GaudiStableVideoDiffusionPipeline):
```

just a personal suggestion to be compliant with other naming like `GaudiStableDiffusionControlNetPipeline`

Reply:
Done, thanks.